script/get-mlperf-inference-src/customize.py

```diff
@@ -45,7 +45,7 @@ def preprocess(i):
             env["CM_GIT_URL"] = "https://github.com/mlcommons/inference"
     
     if env.get("CM_MLPERF_LAST_RELEASE", '') == '':
-        env["CM_MLPERF_LAST_RELEASE"] = "v4.0"
+        env["CM_MLPERF_LAST_RELEASE"] = "v4.1"
```

Comment:
When we use `v4.1-dev` variation (the default) this should be set to v4.0 from the meta.

Reply:
Updated: [624cadf](https://github.com/mlcommons/cm4mlops/pull/353/commits/624cadfba8089856ea2815854cb33b7bf157aa5b)